integrations/extensions/docs/elasticsearch-install-and-setup/text_embedding_deploy_and_use.md

```diff
@@ -0,0 +1,189 @@
+# How to set up and use 3rd-party text embeddings for dense vector search in Elasticsearch
+This guide demonstrates how to deploy and use a text embedding model in Elasticsearch. The model will generate vector representations for text, enabling vector similarity (k-nearest neighbours) search.
+
+Please refer to [this guide](./ICD_Elasticsearch_install_and_setup.md) first to create an Elasticsearch instance and set up Kibana if you haven't already.
+
+## Install the eland library
+Run the command below to install the [eland](https://github.com/elastic/eland) library.
+```bash
+python -m pip install "eland[pytorch]"
```

Comment:
Hi Reza, I think it would be nice to add something more explicit about compatibility. For example, 

NOTE: As of the time this documentation was written, `eland` only supports Python 3.8, 3.9, and 3.10. Please refer to the eland library [compatibility section](https://github.com/elastic/eland?tab=readme-ov-file#compatibility) to make sure you're using compatible Python and Elasticsearch versions. 

Also, adding the note right after this Python command looks more intuitive to me.

Reply:
Good point, thanks! Will add this note